HDSAviz/Interactive_vis_toolkit.ipynb

```diff
@@ -0,0 +1,443 @@
+    "    label_angle[label_angle < -np.pi/2] += np.pi\n",
+    "    # Placing Labels and Legend\n",
+    "    p.text(xr, yr, df.Parameter, angle=label_angle,\n",
+    "           text_font_size=\"9pt\", text_align=\"center\", text_baseline=\"middle\")\n",
+    "\n",
+    "    p.rect([-40, -40], [18, 0], width=30, height=13,\n",
+    "           color=list(stat_color.values()))\n",
+    "    p.text([-15, -15], [18, 0], text=Cols,\n",
+    "           text_font_size=\"9pt\", text_align=\"left\", text_baseline=\"middle\")\n",
+    "\n",
+    "    \n",
+    "    return p\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Code Below is the interactive plots version"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "The current program is meant to draw interactive visualizations using BOKEH\n",
+    "and ipywidgets libraries for sensitivity analysis\n",
+    "\n",
+    "Dependencies : \n",
+    "Currently this code needs to be run Inside Ipython notebook.\n",
+    "Python version :2.7\n",
+    "\n",
+    "Note:\n",
+    "Please ensure Basic_Plot.py exists in same directory and necessary data files are\n",
+    "located in c:/HDSAviz_data folder\n",
+    "\n",
+    "Basically files should be organized as : \n",
+    "\n",
+    "C: HDSAviz_data/\n",
+    "C: HDSAviz_data/analysis_CO.txt \n",
+    "C: HDSAviz_data/analysis_CO2.txt \n",
+    "etc.\n",
+    "\n",
+    "C: HDSAviz_data/interactivevislib.py \n",
+    "C: HDSAviz_data/Interactive_vis_toolkit.ipynb\n",
+    "\n",
+    "\"\"\"\n",
+    "\n",
+    "#import Basic_Plot  once Basic_Plot.py file is ammended.\n",
+    "\n",
+    "import os\n",
+    "from bokeh.models.widgets import Panel, Tabs\n",
+    "from bokeh.io import output_file, output_notebook, show, vform\n",
+    "from ipywidgets import FloatSlider\n",
+    "from IPython.html.widgets import interact\n",
+    "from collections import OrderedDict\n",
+    "from bokeh.plotting import figure, show, output_notebook, output_file\n",
+    "\n",
+    "# initializing the range of values for total order (ST) and first order \n",
+    "# (S1) sensitivity tests \n",
+    "ST_Range = 1\n",
+    "S1_Range = 1\n",
+    "\n",
+    "def change_directory():\n",
+    "    \"\"\"\n",
+    "    This function changes the working directory from C:Users/UserName to\n",
+    "    C:/HDSAviz_data/ which is chosen as convention for current project.\n",
+    "    \n",
+    "    Output: \n",
+    "    Changes working directory and prints current directory\n",
+    "    \"\"\"\n",
+    "    path = \"c:/HDSAviz_data\"\n",
+    "    os.chdir(path)\n",
+    "    print 'current working directory: ' + os.getcwd()\n",
+    "\n",
+    "\n",
+    "def gather_all_analysis_files():\n",
```

Comment:
This function can probably be simplified (or made obsolete) because my function `get_sa()` returns a dictionary with the analysis names as keys, so `dict.keys()` will give the same information as this function.  Wait to edit it until we all go through the new functions in data_processing.py though.